tutorials/text/text_props.py

```diff
@@ -209,6 +209,11 @@
 #    # This is effectively translated to:
 #    matplotlib.rcParams['font.family'] = ['Family1', 'SerifFamily1', 'SerifFamily2', 'Family2']
 #
+# .. note::
+#    To access the full list of available fonts: ::
+#
+#       matplotlib.font_manager.get_font_list()
```

Comment:
I'd move this up to l.177:

```
# Matplotlib can use font families installed on the user's computer, i.e.
# Helvetica, Times, etc. `matplotlib.font_manager.get_font_names()`
# lists all available fonts.
```

Reply:
Thanks for pointing this out! I completely missed those lines. Note moved.